plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -70,7 +73,37 @@ def _coerce_to_cartesian_si(pos):
     return pos_out
 
 
-class Tracker:
+class _SyntheticRadiographySaveRoutine(AbstractSaveRoutine):
+    ARRAY_KEYS = ["x", "y", "v", "x0", "y0", "v0"]
+
+    def __init__(self, output_directory: Path):
+        super().__init__(output_directory=output_directory)
+
+    @property
+    def require_synchronized_dt(self) -> bool:
+        return False
+
+    @property
+    def save_now(self) -> bool:
+        return False
+
+    def save(self) -> None:
```

Comment:
If this is a subclass of AbstractSaveRoutine, then shouldn't it really overwrite `_save_to_disk` and `_save_to_memory`, and leave `save` alone? 

Maybe we should have included a more extendable way of saving quantities in `AbstractSaveRoutine`, say a dict `self.results = {}` For the abstract class, that would have keys `x` and `v`. For the charged particle radiograph class, you could extend it to also save these other variables. Then, `_save_to_disk` would just always save whatever was in the dictionary to the HDF5 file?